cosmo_tester/framework/test_hosts.py

```diff
@@ -736,6 +746,38 @@ def _update_instances_list(self, outputs):
                     self._tmpdir
                     )
 
+    def _save_manager_logs(self, err_raised, purge=True):
+        self._logger.debug('_save_manager_logs started')
+        logs_dir = os.environ.get('CFY_LOGS_PATH_LOCAL')
+        test_path = self._tmpdir.name
+        if not logs_dir:
+            self._logger.debug('not saving manager logs')
```

Comment:
Explain why this did not happen, so the user could fix this with looking the code for the why

Reply:
Changed.